blocks/components/editable/index.js

```diff
@@ -155,14 +156,15 @@ export default class Editable extends wp.element.Component {
 	}
 
 	render() {
-		const { tagName: Tag = 'div', style, className } = this.props;
-		const classes = classnames( 'blocks-editable', className );
+		const { tagName: Tag = 'div', style, className, value, placeholder } = this.props;
+		const classes = classnames( 'blocks-editable', className, { 'is-empty': ! value } );
 
 		return (
 			<Tag
 				ref={ this.bindNode }
 				style={ style }
-				className={ classes } />
+				className={ classes }
+				placeholder={ placeholder } />
```

Comment:
Wouldn't `placeholder` be an invalid attribute for non-input/textarea elements? Maybe `data-placeholder` instead?